### ISSUE TITLE
Kernel version check isn't working

### ISSUE THREAD

**Issue by jwbensley** [jwbensley/EtherateMT#3]
Using `#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,14,0)` from `version.h` doesn't work on certain distros, for example CentOS when using a non-standard Kernel. CentOS 7 uses Kernel 3.10 but installing a 4.13 Kernel for example doesn't update `LINUX_VERSION_CODE` in `/usr/include/linux/version.h` and the EtherateMT checks fail.

```
#include <sys/utsname.h>
#include <stdio.h>

main () {

  struct utsname uts;
  uname(&uts);
  float f;

  sscanf (uts.release,"%f",&f);
  if (f >= 3.14) printf ("%f is >= 3.14\n", f);

   return 0;
}
```

**Comment by jwbensley** [comment]
This instroduces a new problem that the compile will fail as this is a runtime test not a compile time test:
```
packet_mmap.c:442:79: error: ‘PACKET_QDISC_BYPASS’ undeclared (first use in this function)
             int32_t sock_qdisc_ret = setsockopt(thd_opt->sock_fd, SOL_PACKET, PACKET_QDISC_BYPASS, &sock_qdisc_bypass, sizeof(sock_qdisc_bypass));
```

**Comment by jwbensley** [comment]
Replace this:

```
            ;
            static const int32_t sock_qdisc_bypass = 1;
            return setsockopt(thd_opt->sock_fd, SOL_PACKET, PACKET_QDISC_BYPASS, &sock_qdisc_bypass, sizeof(sock_qdisc_bypass));
```

With this:
```
            ;
            #if !defined(PACKET_QDISC_BYPASS)
                return EXIT_SUCCESS;
            #else
                static const int32_t sock_qdisc_bypass = 1;
                return setsockopt(thd_opt->sock_fd, SOL_PACKET, PACKET_QDISC_BYPASS, &sock_qdisc_bypass, sizeof(sock_qdisc_bypass));
            #endif
```

**Comment by jwbensley** [comment]
Fixed in Beta 0.5.
